Harvest.Core/Extensions/DateTimeExtensions.cs

```diff
@@ -18,6 +19,29 @@ private static TimeZoneInfo GetPacificTimeZone()
             }
         }
 
+        public static bool IsBusinessDay(this DateTime dateTime)
+        {
+            return dateTime.ToPacificTime().DayOfWeek switch
+            {
+                DayOfWeek.Saturday => false,
+                DayOfWeek.Sunday => false,
+                _ => true
+            };
+        }
+
+        public static bool IsFirstBusinessDayOfMonth(this DateTime dateTime)
+        {
+            var pacificTime = dateTime.ToPacificTime();
+            var day = pacificTime.DayOfWeek;
+            return day switch
+            {
+                DayOfWeek.Saturday => false,
+                DayOfWeek.Sunday => false,
+                DayOfWeek.Monday => pacificTime.Day <= 3,

```

Comment:
I don't like this because it isn't clear what it is doing. But maybe I'm just too old.

Reply:
It's just shorthand for if/else-if/else-if/else